stdlib/unittest/result.pyi

```diff
@@ -1,9 +1,12 @@
+import sys
 import unittest.case
 from _typeshed import OptExcInfo
 from collections.abc import Callable
 from typing import Any, TextIO, TypeVar
+from typing_extensions import TypeAlias
 
 _F = TypeVar("_F", bound=Callable[..., Any])
+_DurationsType: TypeAlias = list[tuple[unittest.case.TestCase, float]]
```

Comment:
At least `collectedDurations` (ugh, camel case...) uses `list[tuple[str, float]]`:

https://github.com/python/cpython/blob/24b9bdd6eaf0b04667b6cd4c8154f7c7c10c2398/Lib/unittest/result.py#L170

Reply:
Although the docs are wrong: https://docs.python.org/3.12/library/unittest.html?highlight=collecteddurations#unittest.TestResult.collectedDurations

But even the tests use `str`: https://github.com/python/cpython/blob/24b9bdd6eaf0b04667b6cd4c8154f7c7c10c2398/Lib/test/test_unittest/test_runner.py#L1402

I will send a PR to CPython.

Reply:
Yeap, I used docs for this type. Let's wait for your PR to land first.

Reply:
Looks like a deliberate change to fix a circular reference, they just forgot to update the docs: https://github.com/python/cpython/pull/106888